wiki/Mappers_Guild/en.md

```diff
@@ -0,0 +1,49 @@
+
+## History
+
+The Mappers' Guild launched with a group of less than 10 mappers in late 2017. Given information on an upcoming featured artist at the time, [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19), they were tasked with creating 5 beatmaps to accompany the artist's [news post announcement](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid).
+
+To promote older featured artists, the group increased its member count and ranked 8 beatmaps from different artists. These beatmaps were published in a [news post](https://osu.ppy.sh/home/news/2018-03-17-new-mappers-guild-pack-release) alongside a downloadable beatmap pack.
+
+Following in the footsteps of both of these beatmap packs, the Mappers' Guild continuously increased its member count while publishing packs for new featured artist announcements, such as [High Tea Music](https://osu.ppy.sh/home/news/2018-08-13-new-featured-artist-high-tea-music), [Culprate](https://osu.ppy.sh/home/news/2018-11-21-new-featured-artist-culprate), and [HyuN](https://osu.ppy.sh/beatmaps/artists/33), accompanied by other packs for older featured artist songs.
+
+### Automation
+
+While the initial plan for the Mappers' Guild had always been to have MMO elements like quests, that was found to be impractical to handle while manually managing deadlines for the limited number of guild participants.
+
+Development of a website to automate Mappers' Guild began at the end of 2018, allowing for more adequate distribution of rewards and management of quest/party systems.
+
+The [Mappers' Guild website](https://mappersguild.com/) launched on March 1st, 2019, and is where everything about the Mappers' Guild is currently controlled. Since being opened to the public, the guild has reached a member count over 500 and publishes news posts such as [this](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) to keep the rest of the osu! community up to date with featured artist mapping content.
+
+## System
+
+Mappers with 3 or more ranked maps can join the guild at any time via the [Mappers' Guild website](https://mappersguild.com/) by authenticating their osu! account. Anyone else is free to view the website's quest and mapping activities.
+
+Mappers can earn rewards by participating in the ranking process for featured artist maps that are connected to the Mappers' Guild. This primarily focuses on creating beatmaps, however rewards are also provided for users who contribute by modding, creating storyboards, and complete quests within the guild.
+
+### Parties
+
+Parties are groups of mappers who register to complete quests together. Mappers may join as many parties as they want at a time.
+
+Special quests require higher guild ranks to participate in. For a party to qualify as a higher rank, at least half of its members must be the required rank or higher.
+
+### Quests
```

Comment:
you could add a sentence or two describing the new features of how quests cost points to enter and that they may expire after a period of time